ocs_ci/utility/utils.py

```diff
@@ -352,21 +352,42 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
+def mask_secrets(cmd, secrets):
+    """
+    Replace secrets in plaintext with asterisks
+
+    Args:
+        cmd (str): The command/plaintext to remove the secrets from
+        secrets (list): List of secret strings to replace in the cmd
+
+    Returns:
+        cmd (str): The censored version of cmd
+    """
+    for secret in secrets:
+        cmd = cmd.replace(secret, '*'*len(secret))
```

Comment:
I think that a few asterisks is enough.  For example 5 as you will never know the right length of secret which is more secure :)

Reply:
I actually thought about that, thought that secret are fixed-length, but then found out they actually aren't :)
Hoped it'd be good for debugging purposes (for example, secrets that are obviously too long/short), but the security is more important.

Replaced! Important catch. Thank you :)